ivy/functional/frontends/torch/loss_functions.py

```diff
@@ -12,7 +48,12 @@ def cross_entropy(
     reduction="mean",
     label_smoothing=0.0,
 ):
-    return ivy.cross_entropy(input, target)
+    input = ivy.softmax(input)
```

Comment:
Why do we softmax here?

Reply:
The `cross_entropy` function applies LogSoftmax and then NLLLoss. As it is in the implementation details here 
[CROSSENTROPYLOSS](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html#torch.nn.CrossEntropyLoss)
This softmax here is for converting the input to a normalized probability distribution. then we apply our ivy `cross_entropy` loss function.

Reply:
Ok that makes sense